### ISSUE TITLE
Missing AATool.exe

### ISSUE THREAD

**Issue by PalePinkPrince** [DarwinBaker/AATool#6]
Hi! 

So I was planning on using this for my stream since I'm doing a casual AA world, but when I go to set it up I can't seem to find the AATool.exe file. I don't know if it's when I extract the zip file that it gets lost or something else. I've gone through evert file but I just don't see it. I'd really appreciate any advice if you could give some.

**Comment by DarwinBaker** [comment]
Hello!

Did you check your antivirus settings? What antivirus software are you running? Executables disappearing when being unzipped is almost always antivirus software incorrectly thinking it's a virus and removing it without telling you. Depending on what software you have, you may have to add the exe as an "exception" in your antivirus settings so it knows to leave it alone. Let me know if there's anything else I can do to help, and best of luck!

**Comment by PalePinkPrince** [comment]
So,

I allowed it on my Malwarebytes and deleted McAfee entirely and the AATool.exe is still missing. I've tried redownloading it a few times and I still can't see it anywhere in the files.

**Comment by DarwinBaker** [comment]
Is it possible you downloaded the source code instead of the release by mistake? That would also explain there being no exe. 

You want to make sure you're downloading "ctm_aa_tool_1.3.0.1.zip" from https://github.com/DarwinBaker/AATool/releases/latest

Hope this helps!

**Comment by PalePinkPrince** [comment]
I was downloading the wrong file but now i can't open the file. SOrry about this, I guess I'm not really good with this kind of stuff.
![aatoolexe response message](https://user-images.githubusercontent.com/89160180/129976080-82c59ff0-2cbd-41a5-a871-c391a8f99401.PNG)
This comes up when I try to open the AATool.exe

**Comment by PalePinkPrince** [comment]
> I was downloading the wrong file but now i can't open the file. SOrry about this, I guess I'm not really good with this kind of stuff.
> ![aatoolexe response message](https://user-images.githubusercontent.com/89160180/129976080-82c59ff0-2cbd-41a5-a871-c391a8f99401.PNG)
> This comes up when I try to open the AATool.exe

Also I did redownload and I still get this message when I try to open it.

**Comment by DarwinBaker** [comment]
Are you running the executable from inside the zip file? If so, you must first extract the zip file to a folder and then run AATool.exe from there.

**Comment by PalePinkPrince** [comment]
Yeah I've been extracting the zip file into a folder. The only other thing I can open is the AATool file that has everything all together and the updater. There is an AATool.exe.config though, that's probably supposed to be there but I figured I'd bring it up just in case it isn't.

**Comment by DarwinBaker** [comment]
My apologies, but I'm not fully understanding what your folder looks like. Would you mind sending a screenshot? Thanks!

**Comment by PalePinkPrince** [comment]
![this is what they look like](https://user-images.githubusercontent.com/89160180/129980135-351da27f-d092-4b84-a74e-598df486384a.PNG)
This is what the folder for it looks like

**Comment by DarwinBaker** [comment]
I can't see the directory path, are you 100% sure it doesn't say ".zip" anywhere in there? I can't figure out why it says it can't find the update executable, it's right there in the picture of the folder you sent 🤔

**Comment by PalePinkPrince** [comment]
Yeah there isn't a zip name in the directory path. 
![the directory path](https://user-images.githubusercontent.com/89160180/129981880-d1543f94-5192-4a13-8a94-5817cddf9860.PNG)
Also the aatool master is just what i named the big folder it's in, its not got anything else in it

**Comment by DarwinBaker** [comment]
What happens when you run AAUpdate.exe?

**Comment by PalePinkPrince** [comment]
It says that I already have the latest version of it.

**Comment by DarwinBaker** [comment]
New version released, would you mind deleting the "assets" folder and running AAUpdate.exe to see if it works now?
